lib/srv/monitor.go

```diff
@@ -224,6 +230,8 @@ type MonitorConfig struct {
 	ServerID string
 	// Emitter is events emitter
 	Emitter apievents.Emitter
+	// EmitterContext is long-lived context suitable to be used with Emitter.
+	EmitterContext context.Context
```

Comment:
How is this different from the other Context field?

Reply:
This one is long-lived: if the server is not shutting down, we should always attempt to emit the audit event. The other one is short-lived and may be cancelled when we close the connection, which this file is all about.

Reply:
Gotcha. I would note that in the godoc.